bazel/swift_static_framework.bzl

```diff
@@ -43,6 +49,14 @@ def _swift_static_framework_impl(ctx):
             fail("Unhandled platform '{}'".format(platform))
 
         swift_info = archive[SwiftInfo]
+
+        # We can potentially simplify this if this change lands upstream:
+        # https://github.com/bazelbuild/rules_swift/issues/291
+        objc_headers = archive[apple_common.Objc].header.to_list()
+        objc_header = objc_headers[-1]
```

Comment:
I don't know there's a guarantee that the generated header will be last.

Reply:
Perhaps filter for `-Swift.h`, optionally assert that there's only one, and take that header.

Reply:
Sounds good, I'll add that in a separate PR shortly.

Reply:
https://github.com/lyft/envoy-mobile/pull/374